javaparser-core/src/main/java/com/github/javaparser/ast/stmt/CatchClause.java

```diff
@@ -68,6 +69,7 @@ public CatchClause(final Range range,
 		v.visit(this, arg);
 	}
 
+    // TODO Should we deprecate those ?
 	public BlockStmt getCatchBlock() {
```

Comment:
Because of getBody() setBody() ? Or we let them as duplicates doing the same thing (strange)?